.zedstate

```diff
@@ -0,0 +1 @@
":{"start":{"row":21,"column":44},"end":{"row":22,"column":0}},"text":"\n"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":22,"column":0},"end":{"row":22,"column":56}},"text":"app.use(require('morgan')({ \"stream\": logger.stream }));"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":23,"column":0},"end":{"row":23,"column":1}},"text":"/"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":23,"column":1},"end":{"row":23,"column":2}},"text":"/"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":8,"column":7},"end":{"row":8,"column":8}},"text":"g"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":8,"column":8},"end":{"row":8,"column":9}},"text":"e"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":8,"column":9},"end":{"row":8,"column":10}},"text":"r"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":20,"column":54},"end":{"row":21,"column":0}},"text":"\n"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":21,"column":0},"end":{"row":21,"column":36}},"text":"logger.info(\"Listening on \" + port);"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":24,"column":13},"end":{"row":24,"column":14}},"text":"g"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":24,"column":14},"end":{"row":24,"column":15}},"text":"e"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":24,"column":15},"end":{"row":24,"column":16}},"text":"r"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":21,"column":0},"end":{"row":21,"column":1}},"text":"/"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":21,"column":1},"end":{"row":21,"column":2}},"text":"/"}]}],[{"group":"doc","deltas":[{"action":"removeText","range":{"start":{"row":24,"column":1},"end":{"row":24,"column":2}},"text":"/"}]},{"group":"doc","deltas":[{"action":"removeText","range":{"start":{"row":24,"column":0},"end":{"row":24,"column":1}},"text":"/"}]}],[{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":24,"column":0},"end":{"row":24,"column":1}},"text":"/"}]},{"group":"doc","deltas":[{"action":"insertText","range":{"start":{"row":24,"column":1},"end":{"row":24,"column":2}},"text":"/"}]},{"group":"doc","deltas":[{"action":"removeText","range":{"start":{"row":21,"column":1},"end":{"row":21,"column":2}},"text":"/"},{"action":"removeText","range":{"start":{"row":21,"column":0},"end":{"row":21,"column":1}},"text":"/"}]}],[{"group":"doc","deltas":[{"action":"removeText","range":{"start":{"row":21,"column":0},"end":{"row":21,"column":36}},"text":"logger.info(\"Listening on \" + port);"},{"action":"removeText","range":{"start":{"row":20,"column":54},"end":{"row":21,"column":0}},"text":"\n"}]}]],"redo":[]}},"window":{"width":1366,"height":688,"top":0,"left":0,"isMaximized":true}}
```

Comment:
@thinkking remove the `.zedstate` file from the repository and add an entry into the `.gitignore` files.  Also remove the `bower_components` directory from the repo.